ios/FluentUI/Navigation/Shy Header/ShyHeaderView.swift

```diff
@@ -194,12 +194,12 @@ class ShyHeaderView: UIView {
     }
 
     private func initShadow() {
-        shadow.translatesAutoresizingMaskIntoConstraints = false
-        addSubview(shadow)
+        shadow.view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Can you store shadow.view in a constant to be used throughout this implementation?

`let shadowView = shadow.view`